freshmaker/lightblue.py

```diff
@@ -228,6 +229,15 @@ def get_additional_data_from_koji(cls, nvr):
                         "Cannot find task_id or container_koji_task_id "
                         "in the Koji build %r" % build)
 
+            fs_koji_task_id = build.get('extra', {}).get('filesystem_koji_task_id')
+            if fs_koji_task_id:
+                parsed_nvr = koji.parse_NVR(nvr)
+                name_version = f'{parsed_nvr["name"]}-{parsed_nvr["version"]}'
+                if name_version not in conf.image_extra_repo:
+                    msg = (f'Build for {name_version} contains filesystem_koji_task_id, '
```

Comment:
I'd suggest an error message like this:
{name_version} is a base image, extra repo in Freshmaker configuration is missing for building this base image.